### PR TITLE
feat(Button): refactor button css

### PR DESCRIPTION
**Pull request by snorrekim**
* Padding is always used on the "outside" of text and icon
* Margin is always used between text and icon
* Css based on button height and icon size is now calculated using variables

### CODE REVIEW

File: packages/dnb-eufemia/src/components/global-status/__tests__/__image_snapshots__/globalstatus-for-ui-have-to-match-the-close-button-in-focus-state.snap.png
Comment (tujoworker) [inline packages/dnb-eufemia/src/components/global-status/__tests__/__image_snapshots__/globalstatus-for-ui-have-to-match-the-close-button-in-focus-state.snap.png:1]: It appears that the left padding on the button has been altered. Was this change intentional?

**Review by tujoworker — APPROVED** [review]
Really nice work ⭐ 

Left one comment you may check out before merge.

File: packages/dnb-eufemia/src/components/global-status/__tests__/__image_snapshots__/globalstatus-for-ui-have-to-match-the-close-button-in-focus-state.snap.png
Comment (snorrekim) [reply packages/dnb-eufemia/src/components/global-status/__tests__/__image_snapshots__/globalstatus-for-ui-have-to-match-the-close-button-in-focus-state.snap.png:1]: Yes, that's intentional, the tertiary focus effect was too wide compared to the design
